Dashboard

rk-pdx opened the pull request:
- Adding to Contact.jsx, Contact.css
- Added a new component called ContactForm.jsx, as well as its paired CSS file ContactForm.css.
- Changed contactAbout.css to contact.css.
- Added post request to backend route handler
- Added backend route handler to handle post request
- Backend route handler will email the destination email address (WIP - authentication failure).

rk-pdx committed:
Merge pull request #10 from rk-pdx/PostReplies
- In PostCompact.jsx, changed the title to have the same functionality as the RepliesBtn, so that when it is clicked, the PostModal will open.
- Added styles to the above button.
- Removed the save button
- Changed style to createPostModal.jsx
- Changed background color of navigation bar
- Modified form and modal functionality, including styling, in CreatePostModal.jsx.
- Removed the body from showing on PostCompact
- Changed the category field from optional to required
- Updated createPostModal to close upon successful submission of post data

zfreiter committed:
Merge branch 'main' into dashboard

CristianTS599 reviewed (commented):
looks good just fix the merge conflicts and you should be good to go

rk-pdx committed:
Testing

rk-pdx committed:
- Changed styling on CreatePostModal.jsx
- Reverted back some Featued Posts code so that original isn't broken.
- changed styling on toolbar, including dashboard's create post button.
